packages/server/src/rewire.ts

```diff
@@ -5,7 +5,11 @@ if (process.env.TS_NODE_DEV) {
   const originalRequire = Module.prototype.require as (id: string) => any
 
   const rewire = function (this: (id: string) => any, mod: string) {
-    if (mod.startsWith('@botpress')) {
+    if (
+      mod.startsWith('@botpress/messaging') &&
+      // we don't want to rewire the legacy channels because we get them from npm
+      mod !== '@botpress/messaging-channels-legacy'
+    ) {
       return originalRequire.apply(this, [mod + '/src'])
     }
```

Comment:
Had to add this otherwise ts-node would try to run the legacy channel code from source